src/actions/types.js

```diff
@@ -0,0 +1,2 @@
+export const TEST = "TEST";
```

Comment:
Single quote over double quote
```suggestion
export const TEST = 'TEST';
```